placement-rules-in-sql.md

```diff
@@ -0,0 +1,154 @@
+ALTER DATABASE test FOLLOWERS=2; -- this does not apply to existing tables
+CREATE TABLE t4 (a INT); -- the table is created with FOLLOWERS=2;
+```
+
+Because placement options are only inherited from the database schema default when a table is created, it is recommended to set the default to a `PLACEMENT POLICY`. This ensures that future changes to the policy will propagate to existing tables.
+
+### Advanced placement
+
+The placement options `PRIMARY_REGION`, `REGIONS` and `SCHEDULE` are designed to support the primary use cases of data placement at the loss of some flexibility. For more complex scenarios it is possible to also use the advanced placement options of `CONSTRAINTS` and `FOLLOWER_CONSTRAINTS`. Use of these optons is mutually exclusive, and an error will be returned if both are specified.
+
+For example, to set constraints that data must reside on a TiKV store where the label `disk` must match a value:
+
+```sql
+CREATE PLACEMENT POLICY storeonfastssd CONSTRAINTS="[+disk=ssd]";
+CREATE PLACEMENT POLICY storeonhdd CONSTRAINTS="[+disk=hdd]";
+CREATE PLACEMENT POLICY companystandardpolicy CONSTRAINTS="";
+
+CREATE TABLE t1 (id INT, name VARCHAR(50), purchased DATE)
+PLACEMENT POLICY=companystandardpolicy
+PARTITION BY RANGE( YEAR(purchased) ) (
+  PARTITION p0 VALUES LESS THAN (2000) PLACEMENT POLICY=storeonhdd,
+  PARTITION p1 VALUES LESS THAN (2005),
+  PARTITION p2 VALUES LESS THAN (2010),
+  PARTITION p3 VALUES LESS THAN (2015),
+  PARTITION p4 VALUES LESS THAN MAXVALUE PLACEMENT POLICY=storeonfastssd
+);
+```
+
+Constraints can either be specified in list format (`[+disk=ssd]`) or dictionary format (`{+disk=ssd:1,+disk=hdd:2}`).
+
+In list format, constraints are specified as a list of key-value pairs. The key starts with either a `+` or a `-` with `+disk=ssd` indicating that the label `disk` must be set to `ssd`, and `-disk=hdd` indicating that the label `disk` must not be `hdd`.
+
+In dictionary format, the constraint also indicates a number of instances that apply to that rule. For example `FOLLOWER_CONSTRAINTS="{+region=us-east-1:1,+region=us-east-2:1,+region=us-west-1:1,+any:1}";` indicates that 1 follower is in us-east-1, 1 follower is in us-east-2, 1 follower is in us-west-1, and 1 follower can be in any region.
+
+## Known limitations
+
+The following known limitations exist in the experimental release of Placement Rules in SQL:
+
+* Dumpling does not support dumping placement policies. See [issue #29371](https://github.com/pingcap/tidb/issues/29371)
+* TiDB tools including BR, CDC, Lightning and DM do not yet support placement rules.
+* Temporary tables do not support placement options (either via direct placement or placement policies).
+* Syntactic sugar rules exist for setting `PRIMARY_REGION` and `REGIONS`, but in future we plan to add varieties for `PRIMARY_RACK`, `PRIMARY_ZONE` and `PRIMARY_HOST`. See [issue #18030](https://github.com/pingcap/tidb/issues/18030)
```

Comment:
Yes, but this is under the headline "Known limitations" as it applies to the experimental release. This is a known limitation, since there are other configurations besides regionality.

Reply:
Get it. I've modified my comment. PTAL